.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build-push:
+    name: 📦 Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: docker/setup-buildx-action@f7ce87c1d6bead3e36075b2ce75da1f6cc28aaca # v3.9.0
+      - uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        with:
+          registry: ghcr.io
```

Comment:
Για την περίπτωση που πάμε στην λογική του app platform και για το bot, μια σημείωση εδώ είναι πως βάσει του [app spec](https://docs.digitalocean.com/products/app-platform/reference/app-spec/) το `deploy_on_push` για τα container image apps λειτουργεί αυτόματα μόνο για το docker.io. Άρα αν θέλουμε να έχουμε και CD ίσως θα μπορούσαμε είτε να αλλάξουμε registry είτε να πάμε στη λογική του [app platform github action](https://www.digitalocean.com/blog/github-actions-for-app-platform).

Reply:
Σωστό. Είχα σκεφτεί να πάμε αρχικά με manual deployments για να τεστάρουμε οτι ολα λειτουργούν σωστά και μετά να ανοίξω ένα ακόμη PR με τα updates για automatic deployments.

Reply:
Να κάναμε ένα issue να έχει scope το ιδιο task και στο website και στο bot? Γιατί και κει έχουμε πει να το δούμε cc @gmargaritis

Reply:
Yes.